contracts/BaseJumpRateModelV2.sol

```diff
@@ -102,38 +102,43 @@ abstract contract BaseJumpRateModelV2 is InterestRateModel {
      * @param borrows The amount of borrows in the market
      * @param reserves The amount of reserves in the market
      * @param reserveFactorMantissa The current reserve factor for the market
+     * @param badDebt The amount of badDebt in the market
      * @return The supply rate percentage per block as a mantissa (scaled by EXP_SCALE)
      */
     function getSupplyRate(
         uint256 cash,
         uint256 borrows,
         uint256 reserves,
-        uint256 reserveFactorMantissa
+        uint256 reserveFactorMantissa,
+        uint256 badDebt
     ) public view virtual override returns (uint256) {
         uint256 oneMinusReserveFactor = MANTISSA_ONE - reserveFactorMantissa;
-        uint256 borrowRate = _getBorrowRate(cash, borrows, reserves);
+        uint256 borrowRate = _getBorrowRate(cash, borrows, reserves, badDebt);
         uint256 rateToPool = (borrowRate * oneMinusReserveFactor) / EXP_SCALE;
-        return (utilizationRate(cash, borrows, reserves) * rateToPool) / EXP_SCALE;
+        uint256 rate = ((borrows * EXP_SCALE) / (cash + borrows + badDebt - reserves));
```

Comment:
I think we can avoid the `/ EXP_SCALE` in the `rateToPool` and the `* EXP_SCALE` in the `rate` calculations, because they are "cancelled" when we multiply them

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/225#discussion_r1209240375